code/controllers/subsystem/garbage.dm

```diff
@@ -234,14 +237,34 @@ SUBSYSTEM_DEF(garbage)
 	queue[++queue.len] = list(gctime, refid) // not += for byond reasons
 
 //this is mainly to separate things profile wise.
-/datum/controller/subsystem/garbage/proc/HardDelete(datum/D)
+/datum/controller/subsystem/garbage/proc/HardDelete(datum/D, non_datum = FALSE, override = FALSE)
```

Comment:
```suggestion
/datum/controller/subsystem/garbage/proc/HardDelete(datum/D, is_datum = TRUE, override = FALSE)
```
Affirmative vars > negative vars. Avoids the `!not_thing` double-negative notation.